### ISSUE TITLE
405 Method Not Allowed status codes

### ISSUE THREAD

**Issue by rjoydip-zz** [medleyjs/medley#1]
I have checked as per your comment [link](https://github.com/fastify/fastify/issues/917#issuecomment-385700136). It doesn't returning any response and not giving any error
reference: [fastify:issue:917](https://github.com/fastify/fastify/issues/917)

**Comment by nwoltman** [comment]
Do you have a test case for this? (It doesn't have to be minimal)

This works for me:

```js
const medley = require('@medley/medley');
const app = medley();

app.route({
  method: ['GET', 'POST'],
  url: '/graphql',
  handler: (req, res) => {
    res.send('Hello World');
  },
});

app.inject({
  method: 'PUT',
  url: '/graphql',
}, (err, res) => {
  if (err) {
    throw err;
  }

  console.log(res.statusCode, res.headers.allow);
  // Prints:
  // 405 'GET,POST,HEAD'
});
```

(HEAD is included because HEAD requests are automatically handled for routes with a GET handler.)

**Comment by rjoydip-zz** [comment]
I was trying by `app.get('/', ...)`  and accessing '/' post routes via `POSTMAN`. But there is not post routes like get.

**Comment by nwoltman** [comment]
Oh, it may be that the body for 405 responses is empty. I've been meaning to fix that 😅

**Comment by nwoltman** [comment]
@rjoydip I just published version `0.6.0`. I'm hoping that will fix your issue.

**Comment by nwoltman** [comment]
Closing since this should be fixed in the latest version. Please re-open if this is still an issue.
